### DISCUSSION TITLE
How to get `pylint` work with virtual environment?

### DISCUSSION THREAD

**Discussion by failable** [python-lsp/python-lsp-server#254] [Q&A]
When I install `python-lsp-server` with `pipx`, then `pylint` plugin always look for imports in the venv of `python-lsp-server`, but not of current vev. How can I get `pylint` work with current venv? Jumping to definitions works just well in venv and I don't need to setup for `pylsp.plugins.jedi.environment`. Even I set, `pylint` still prints lots of `E0401  [import-error] Unable to import 'tqdm'`.

**Comment by ccordoba12** [comment]
Hey @liebkne, this is not supported at the moment in PyLSP, sorry.
